tools/tox_tools/dose_responses/dose_response.xml

```diff
@@ -21,22 +21,28 @@
             '$response_column'
             '$plot_output'
             '$ec_output'
+            '$compound_name'
+            '$concentration_unit'
         ]]>
     </command>
     <inputs>
         <param name="input_csv" type="data" format="tabular" label="Concentration - Response Tabular Input"/>
         <param name="concentration_column" type="data_column" data_ref="input_csv" label="Concentration Column" help="Name of the column for concentration values"/>
         <param name="response_column" type="data_column" data_ref="input_csv" label="Response Column" help="Name of the column for response values"/>
+        <param name="compound_name" type="text" label="Compound Name" help="Name of the compound to analyze"/>
+        <param name="concentration_unit" type="text" label="Concentration Unit"/>
```

Comment:
Should we add more strict sanitizers to these text inputs? Maybe just alphanum + space is sufficient (plus maybe dash, underscore and some punctuation)?